requirements.txt

```diff
@@ -9,8 +9,8 @@
 # that are dependencies of libraries listed in the next section
 
 ga4gh-common==0.0.5
-ga4gh-schemas==0.0.8
-ga4gh-client==0.0.3
+ga4gh-schemas
```

Comment:
I really don't like this.  This means if we release a version of schemas that is incompatible with server (of any version that doesn't pin its requirements exactly), we will be breaking those installs.  We should always exactly pin versions of ga4gh code in requirements files, in my view.

Are you doing this because the constraints feature will not work otherwise?  If so, we have at least a couple options:

1) Have people that want to install with the constraints file (that is, only developers) edit the pins out of the requirements file and then do another pip install with the constraints.  This seems acceptable, since they're editing pip-related files (i.e. `constraints.txt`) anyways

2) Create another requirements file which doesn't have ga4gh versions pinned which can be used with the constraints files